.eslintrc.json

```diff
@@ -2,14 +2,22 @@
   "parser": "babel-eslint",
   "plugins": [
     "flowtype",
-    "import"
+    "import",
+    "react"
   ],
   "extends": ["wellcome"],
   "rules": {
     "no-multi-spaces": "off",
     "no-multi-str": "off",
     "flowtype/define-flow-type": "warn",
     "no-duplicate-imports": "off",
-    "import/no-duplicates": "error"
+    "import/no-duplicates": "error",
+    "react/jsx-uses-react": "error",
```

Comment:
Initially, I've added the two rules that we need to get linting to pass, but I guess we might consider aligning with some sensible defaults before we get too far into moving everything over to JSX: we could use the plugin's [recommended rules](https://www.npmjs.com/package/eslint-plugin-react)

Reply:
Yes, let's get that onto the Reactify project board high priority?